src/Files.Uwp/DataModels/NavigationControlItems/DriveItem.cs

```diff
@@ -143,16 +139,15 @@ public bool ShowStorageSense
         public DriveItem()
         {
             ItemType = NavigationControlItemType.CloudDrive;
+            PropertyChanged += SizeChanged;
```

Comment:
No need for hooking the PropertyChanged event here, instead make a separate method that is called from `SpaceUsed { set; }` and `Path { set; }` and update `HoverDisplayText` from there.

Attaching a PropertyChanged event like that and not deattaching it will result in a memory leak

Reply:
Ok, I didn't know it. Thank you for tips